src/box/index.cc

```diff
@@ -165,11 +175,15 @@ index_point_lookup_validate(struct index_def *index_def, const char *key,
 {
 	if (!index_def->opts.is_unique) {
 		diag_set(ClientError, ER_MORE_THAN_ONE_TUPLE);
-		return -1;
+		error_set_key_with_part_count(key, part_count);
```

Comment:
I don't think we need to know the key for this kind of error.

Reply:
Fixed. Source diff:

```diff
--- a/src/box/index.cc
+++ b/src/box/index.cc
@@ -175,7 +175,6 @@ index_point_lookup_validate(struct index_def *index_def, const ch>
 {
        if (!index_def->opts.is_unique) {
                diag_set(ClientError, ER_MORE_THAN_ONE_TUPLE);
-               error_set_key_with_part_count(key, part_count);
                goto error;
        }
```